### PR TITLE
PivotGrid - Replace requires to ES6 imports

### PR DESCRIPTION
**Pull request by jtoming830**

### CODE REVIEW

File: js/ui/pivot_grid/ui.pivot_grid.field_chooser_base.js [hunk 1]
```diff
@@ -335,7 +338,7 @@ var FieldChooserBase = Widget.inherit(columnStateMixin).inherit(sortingMixin).in
                                     return userData.store.load(options);
                                 } else {
                                     var d = new Deferred();
-                                    dataSource.getFieldValues(mainGroupField.index, that.option("headerFilter.showRelevantValues"), paginate ? options : undefined).done(function(data) {
+                                    dataSource.getFieldValues(mainGroupField.index, that.option("showRelevantValues"), paginate ? options : undefined).done(function(data) {
```
Comment (vconst) [inline js/ui/pivot_grid/ui.pivot_grid.field_chooser_base.js]: Why headerFilter.showRelevantValues is changed to showRelevantValues?

File: js/ui/pivot_grid/data_source.js [hunk 2]
```diff
@@ -1,34 +1,31 @@
...
+import { when, Deferred } from "../../core/utils/deferred";
+import Class from "../../core/class";
+import EventsMixin from "../../core/events_mixin";
+import { titleize } from "../../core/utils/inflector";
+import { normalizeIndexes } from "../../core/utils/array";
+import { LocalStore } from "./local_store";
+import RemoteStore from "./remote_store";
+import { XmlaStore } from "./xmla_store/xmla_store";
+import { applyDisplaySummaryMode, createMockSummaryCell, applyRunningTotal } from "./ui.pivot_grid.summary_display_modes";
+import { foreachTree,
```
Comment (vconst) [inline js/ui/pivot_grid/data_source.js]: Move foreachTree to new line

**Review by vconst — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets.pivotGrid/summaryDisplayModes.tests.js [hunk 3]
```diff
@@ -1,71 +1,69 @@
...
+        index: 1,
+        value: "1",
+        children: [{
+            index: 2,
+            value: "11"
         }, {
-            index: 5,
-            value: "2"
+            index: 3,
+            value: "12"
```
Comment (roman-simionov) [inline testing/tests/DevExpress.ui.widgets.pivotGrid/summaryDisplayModes.tests.js:14]: Why did you change this data item?

Comment (vconst) [reply testing/tests/DevExpress.ui.widgets.pivotGrid/summaryDisplayModes.tests.js:14]: This data item is not changed. Diff tool works bad.

**Review by vconst — DISMISSED** [review]

**Review by vconst — APPROVED** [review]
